blocks/header/header.css

```diff
@@ -1,3 +1,7 @@
+:root {
```

Comment:
Nit: Kindly remove redundant styles from this file which i guess were only relevant for sunstar-engineering
Also some of the styles which u have added also seems to be redundant. Kindly also check those